src/main/java/core/basesyntax/ColorSuplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSuplier {
+    private static String[] colors = new String[] {"black", "white", "yellow",
+            "blue", "red", "pink", "green", "gray"};
+
+    public static String randomColor() {
```

Comment:
```suggestion
    public String randomColor() {
```
don't use static methods - let's create a field of type ColorSupplier in FigureSupplier class